dipy/reconst/brainsuite_shore.py

```diff
@@ -0,0 +1,668 @@
+from __future__ import division
+import warnings
+from math import factorial
+import numpy as np
+from scipy.special import genlaguerre, gamma, hyp2f1
+from dipy.reconst.cache import Cache
+from dipy.reconst.multi_voxel import multi_voxel_fit
+from .shm import real_sym_sh_brainsuite
+from dipy.core.geometry import cart2sphere
+from dipy.utils.optpkg import optional_package
+
+cvxpy, have_cvxpy, _ = optional_package("cvxpy")
+sklearn, have_sklearn, _ = optional_package("sklearn")
+if have_sklearn:
```

Comment:
what happens if there is no `sklearn`, whole fitting crashes? there is no alternative assignment of these functions...